vm/systemSmartContracts/esdt.go

```diff
@@ -2177,8 +2181,11 @@ func (e *esdt) EpochConfirmed(epoch uint32, _ uint64) {
 	e.flagRegisterAndSetAllRoles.SetValue(epoch >= e.registerAndSetAllRolesEnableEpoch)
 	log.Debug("ESDT register and set all roles", "enabled", e.flagRegisterAndSetAllRoles.IsSet())
 
+	e.flagCheckMetaESDTOnRolesEnableEpoch.SetValue(epoch >= e.checkMetaESDTOnRolesEnableEpoch)
```

Comment:
where is this flag used?

Reply:
again - github is stupid. this is already in on master.